db/python/layers/sample.py

```diff
@@ -14,6 +45,7 @@ class SampleLayer(BaseLayer):
     def __init__(self, connection: Connection):
         super().__init__(connection)
         self.st: SampleTable = SampleTable(connection)
+        self.seqt: SampleSequencingTable = SampleSequencingTable(connection)
```

Comment:
Can you move this one into the upsert_sample, if it was used in more endpoints we could promote it up, but for now I think just leave down below.